robot/rospackages/src/task_handler/srv/HandleTask.srv

```diff
@@ -1,4 +1,5 @@
 string task
 int64 status
+string args
 ---
 string response
```

Comment:
it might be cool to have something a bit more clear, like it responds with:
{boolean istaskrunning,
boolean succeeded,
string response}
or maybe even includes an argument confirming which camera stream is on.